web/src/games/bingo/utils.ts

```diff
@@ -17,3 +17,22 @@ export function inferActivePlayers(G: IGameState, playerID: string) {
   const shoutCallPlayers = Object.keys(G.players).filter((p) => G.players[p].shoutCount > 0 && p !== playerID);
   return [playerID, ...shoutCallPlayers].slice(0, MAX_BACKOFF_CANDIDATES);
 }
+
+export function gameLocalStore(action: string, matchCode: string, key: string, value: any = null) {
```

Comment:
Passing a string on `action` to define the behavior of the function is a bad idea, given that the compiler cant validate it is a valid value and typos might lead to unexpected behavior.

I recommend using two functions, one for setting and another for getting in this case.

Reply:
done in new commit.